packages/suite-web/e2e/tests/onboarding/analytics-consent.test.ts

```diff
@@ -24,7 +22,7 @@ const acceptAnalyticsConsentOnInitializedDevice = () => {
 describe('Onboarding - analytics consent', () => {
     const startEmuOpts = {
         url: 'https://gitlab.com/satoshilabs/trezor/trezor-firmware/-/jobs/3104755066/artifacts/raw/core/build/unix/trezor-emu-core',
-        model: DeviceModel.T2B1,
+        model: 'R', // TODO: T2B1 DeviceModelInternal
```

Comment:
waiting for trezor-user-env, right? do we have an issue?

Reply:
In progress, mentioned in todos